components/tool/taskManager/README.md

```diff
@@ -0,0 +1,34 @@
+# ToolTaskManager
```

Comment:
Documentation under construction. It has not been part of the PoC. It will be available during the next weeks.